frontend/ui/network/manager.lua

```diff
@@ -60,6 +61,25 @@ function NetworkMgr:promptWifiOff(complete_callback)
     })
 end
 
+function NetworkMgr:turnOnWifiAndWaitForConnection(callback)
+    NetworkMgr:turnOnWifi()
+    local timeout = 60;
+    local retry_count = 0;
+    local info = InfoMessage:new{ text = T(_("Enabling Wi-Fi. Waiting for Internet connection...\nTimeout %1 seconds."), timeout)}
```

Comment:
Also, true ellipses. `…`

Reply:
I don't understand. Can you clarify?

Reply:
Three periods/dots/points/whatever you want to call them (`...`) is something else than the dedicated ellipsis character (`…`).